GeeksCoreLibrary/Core/Services/WiserItemsService.cs

```diff
@@ -2424,14 +2424,20 @@ public async Task<List<WiserItemModel>> GetLinkedItemDetailsAsync(IWiserItemsSer
                 where.Add("(permission.id IS NULL OR (permission.permissions & 1) > 0)");
             }
 
+            var itemIdTablePrefix = "";
+            if (linkSettings.UseItemParentId && reverse && !string.IsNullOrWhiteSpace(itemIdEntityType))
+            {
+                itemIdTablePrefix = await wiserItemsService.GetTablePrefixForEntityAsync(itemIdEntityType);
+            }
+            
             var tablePrefix = "";
-            if (!String.IsNullOrWhiteSpace(entityType))
+            if (!String.IsNullOrEmpty(entityType))
```

Comment:
I believe this was my mistake since I was refering to String(uppercase) instead of string(no upper case) and copy pasted IsNullOrEmpty from some other place in code instead of the IsNullOrWhiteSpace

Now the String is correct but we use the wrong compare

Reply:
Changed back IsNullOrWhiteSpace to IsNullOrEmpty.